src/generic-container.test.ts

```diff
@@ -160,6 +160,30 @@ describe("GenericContainer", () => {
     await container.stop();
   });
 
+  it("should set log config", async () => {
+    const containerName = "special-test-container";
```

Comment:
Can remove the name expectations, the assertion we care about is on https://github.com/testcontainers/testcontainers-node/pull/58/files#diff-9c365222db7442c7a59145d8c7b621e2R179